FBSnapshotTestCase/FBSnapshotTestCase.m

```diff
@@ -11,17 +11,35 @@
 #import <FBSnapshotTestCase/FBSnapshotTestCase.h>
 #import <FBSnapshotTestCase/FBSnapshotTestController.h>
 
+@interface FBSnapshotTestCase ()
+  @property (nonatomic, strong, nonnull) NSString *folderName;
+@end
+
 @implementation FBSnapshotTestCase
 {
   FBSnapshotTestController *_snapshotController;
 }
 
 #pragma mark - Overrides
 
+- (instancetype)initWithInvocation:(NSInvocation *)invocation
```

Comment:
This works in your testing? This initializer is always called when a test case subclass is first initialized?

Reply:
Yes.